### DISCUSSION TITLE
Query table media has something wrong?

### DISCUSSION THREAD

**Discussion by MNhat0308** [awcodes/filament-curator#521] [Q&A] [answered]
Php: 8.3.8
Laravel version: 11.9.2
Curator: lasted

I have store file successfully but when i edit it have some thing wrong with the query
![image](https://github.com/user-attachments/assets/3507c242-89f6-49a6-bcc1-a25d94e27bcf)

Here is the code i'm using

![Screenshot 2024-08-09 at 22 40 00](https://github.com/user-attachments/assets/9a2eb08c-b07d-490c-b411-dff4f206780d)

Here is the property of the column in database, I have a table name `tenants` too 
![Screenshot 2024-08-09 at 22 41 48](https://github.com/user-attachments/assets/c87d496e-a5db-494a-ae76-b8c5d551e424)

**Comment by MNhat0308** [comment, marked as answer]
[Read more](https://github.com/awcodes/filament-curator/issues/409#issuecomment-1883035774)
So i have find it
we must cast the column to array
![Screenshot 2024-08-09 at 22 53 37](https://github.com/user-attachments/assets/7bea52c3-412e-4d59-b53c-a7ea8539e3d6)
Do you think we should update the readme for this ?
@awcodes

**Reply by awcodes** [reply]
I would accept a PR to update the readme, but I feel like it's general knowledge that if you are saving anything 'multiple' to a single column that it has to be cast as an array. :)

**Reply by MNhat0308** [reply]
I will open a PR for this , thanks for answer
